python/paddle/fluid/layers/nn.py

```diff
@@ -7580,3 +7581,74 @@ def hash(input, hash_size, num_hash=1, name=None):
         attrs={'num_hash': num_hash,
                'mod_by': hash_size})
     return out
+
+
+@templatedoc()
+def grid_sampler(x, grid, name=None):
+    """
+    It sample input X by grid gennerate by AffineGridOp. The grid of shape
+    [N, H, W, 2] is the concatenation of (x, y) coordinates with shape 
+    [N, H, W] each, with x indexing the 4th-D(W) of input feature map and y to 
+    indexng the 3rd-D(H), finally results is the bilinear interpolation value
```

Comment:
y to indexng the 3rd-D(H) -> y is indexing the 3rd dimension (in height direction)